package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "groupby-api",
-  "version": "2.6.2",
+  "version": "2.7.0",
```

The release process will take care of this.
```suggestion
  "version": "2.6.2",
```